### PR TITLE
Molar mass calculation

### PR DESCRIPTION
**Pull request by samuelshaner**
This PR implements a molar mass calculation for a Python API material. 

Additionally, two other issues were addressed:
* Fixed enrichment in input file.
* Added calculation of atomic masses for isotopes that represent all natural isotopes of an element (e.g C0).

### CODE REVIEW

File: openmc/material.py [hunk 1]
```diff
@@ -581,6 +581,35 @@ def get_nuclide_densities(self):
 
         return nuclides
 
+    def get_molar_mass(self):
```
Comment (smharper) [inline openmc/material.py]: I would prefer a `molar_mass` property rather than a getter.

File: openmc/data/data.py [hunk 2]
```diff
@@ -149,6 +149,18 @@ def atomic_mass(isotope):
...
+        # For the isotopes representing all natural isotopes of their element
+        # (e.g. C0), set atomic mass manually using the values from Atomic
+        # weights of the elements 2013 (IUPAC Technical Report)
+        # (doi:10.1515/pac-2015-0305). In cases where an atomic mass range is
+        # given (e.g. C), the average value is used.
+        _ATOMIC_MASS['c0']  = 12.0106
+        _ATOMIC_MASS['zn0'] = 65.38
+        _ATOMIC_MASS['pt0'] = 195.084
+        _ATOMIC_MASS['os0'] = 190.23
+        _ATOMIC_MASS['tl0'] = 204.3835
```
Comment (paulromano) [inline openmc/data/data.py]: Any reason why we shouldn't just use `atomic_mass` and `NATURAL_ABUNDANCE` to just calculate these on the fly?

Comment (samuelshaner) [inline openmc/data/data.py]: Good idea, I'll do that.

**Comment by samuelshaner** [comment]
@paulromano and @smharper - thanks for the quick review. I addressed your comments.

File: openmc/material.py [hunk 3]
```diff
@@ -64,6 +64,12 @@ class Material(object):
         List in which each item is a 3-tuple consisting of an
         :class:`openmc.Nuclide` instance, the percent density, and the percent
         type ('ao' or 'wo').
+    molar_mass : float
+        The molar mass of the material computed in units of grams per mole of
+        nuclides in a material. This entails that the molar mass does not depend
+        on the magnitude of the sum of atomic amounts of elements and nuclides
+        in the material. For instance, the molar mass of UO2 would be
+        ~90 g/mol.
```
Comment (smharper) [inline openmc/material.py]: This explanation seems kind of confusing to me.  I would recommend something more along the lines of:
```
    molar_mass : float
        The average molar mass of nuclides in the material in units of grams per
        mol.  For example, UO2 with 3 nuclides will have an average molar mass
        of 270 / 3 = 90 g / mol.
```
Also along that line, how do you feel about changing the name of the property to `average_molar_mass`?

Comment (samuelshaner) [inline openmc/material.py]: I agree - `average_molar_mass` seems more fitting. I changed the description to match yours as well.

**Pull request merged**